python/views/header.tpl

```diff
@@ -25,6 +26,7 @@
         <li><a href="/">Home</a></li>
         <li><a href="/remote">Remote</a></li>
         <li><a href="/network">Network Setup</a></li>
+        <li><a href="/gps">GPS Setup</a></li>
```

Comment:
I'd rather have this on the 'Tools' page as it should (ideally) be rarely used.  I know it's come up a lot in the discord (people not using GPS, issues with locks), but I hope these are not a representative sample of users. If people are using this often I have failed in some significant way as it's absolutely NOT the intended (or a good) user experience.

Reply:
then I'd need to integrate it with the Tools page, and it's a lot of code, tools might become a garbage bin then. Maybe not adding it to the menu is enough? On save it returns to home

Reply:
Yes... tools is probably not the right place.  How about not on the menu, but accessible from the main page via the edit icon?

Reply:
that's what I did in the last commit